pkg/registry/interface.go

```diff
@@ -17,6 +17,12 @@ type Load interface {
 	ClearNonHeadBundles() error
 }
 
+type BundleCursor interface {
+	Next(context.Context) *api.Bundle
+	Close()
+	Err() error
+}
+
```

Comment:
is the idea here that we need more client-side control than we get from `<-chan *api.Bundle`?